src/pages/cms/index.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { NextPage } from "next";
+import CmsContent from "src/pageContent/cms";
+import Page from "src/components/Page";
+
+const Cms: NextPage = () => {
+  return (
+    <Page
```

Comment:
Just one minor thing, I think better to make it authenticated path, just unsafe kalo semua orang bisa access cms nya